src/libecalc/presentation/yaml/mappers/component_mapper.py

```diff
@@ -265,12 +266,21 @@ def from_yaml_to_dto(self, data: Dict) -> dto.Installation:
             for fuel_consumer in data.get(EcalcYamlKeywords.fuel_consumers, [])
         ]
 
+        if not fuel_consumers and not generator_sets:
+            raise EcalcError(
+                title="Keywords are missing",
+                message=f"It is required to specify at least one of the two keywords "
+                f"{EcalcYamlKeywords.fuel_consumers} or {EcalcYamlKeywords.generator_sets} "
+                f"in the model.",
+            )
```

Comment:
Can we validate this in dto.Installation instead? Shouldn't a model with only venting emitters be allowed?

Reply:
Good point. The manual says that at least one of the keywords must be defined (FUELCONSUMERS or GENERATORSETS). And it fails if both are absent, in graph_result (when subcomponents is empty). Not sure how much work it is to allow only venting emitters.

Reply:
We should probably fix that, but it can of course be a separate issue

Reply:
Agree, I take it in a separate task.